dev/tests/integration/testsuite/Magento/Translation/Model/Js/DataProviderTest.php

```diff
@@ -52,12 +56,11 @@ protected function setUp(): void
      */
     public function testGetData()
     {
-        $expectedDictionary = ['Proceed to Checkout' => 'Proceed to Checkout - Translated'];
-
-        $this->stringUtils->saveTranslate('Proceed to Checkout', 'Proceed to Checkout - Translated');
+        $this->stringUtils->saveTranslate(static::STRING_TO_TRANSLATE, static::STRING_TRANSLATION);
```

Comment:
In my opinion, there is no need to `static` call the const. Let's use `self` as replacement. What do you think?
```suggestion
        $this->stringUtils->saveTranslate(self::STRING_TO_TRANSLATE, self::STRING_TRANSLATION);
```